### ISSUE TITLE
Black Market crashes the game

### ISSUE THREAD

**Issue by lutzee** [WakingStones/CallersBane-Issues#3]
**Describe the bug**
On some servers, trying to use the black market will crash the game.

**To Reproduce**
1. Open black market
2. Game Crashes

**Expected behavior**
Game does not crash and shows the black market

**Server or Client?**
- Client

**Comment by kbasten** [comment]
Fixed, issue caused by mysql settings

**Comment by lutzee** [comment]
Can you explain what was wrong so others can fix it if they have the same problem?

**Comment by kbasten** [comment]
Yes, this is due to the setting ONLY_FULL_GROUP_BY active by default in mysql 5.7+. You can turn this setting off by running the following query:
`SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));`
This does however have the effect of the black market prices not being deterministic in the overview. This is why prices might differ in the black market list and the actual buy panel. There is no fix for that at the moment.

**Comment by AndrewSav** [comment]
Note that `SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));` does not survive reboot. I ended up [specifying sql-mode on the command line](https://github.com/AndrewSav/CallersBane-Docker/blob/master/example/docker-compose.yml#L8) when starting mysql to solve it completely.
